python/lsst/ts/wep/task/calcZernikesNeuralTask.py

```diff
@@ -81,6 +75,12 @@ class CalcZernikesNeuralTaskConnections(
         storageClass="AstropyQTable",
         name="zernikes",
     )
+    donutStamps = connectionTypes.Output(
+        doc="Donut stamps",
+        dimensions=("visit", "detector", "instrument"),
+        storageClass="AstropyQTable",
+        name="donutStamps",
```

Comment:
Let's differentiate these by calling them `donutStampsNeural` or something like that.